projects/sandbox/pyproject.toml

```diff
@@ -187,7 +187,7 @@ spacing = 48
 buffer = 4
 jitter = 2
 cosmology = "${base.cosmology}"
-
+hopeless_snr_threshold = 6
```

Comment:
Have we confirmed that this is the typical threshold? Deep was saying he thought it might be 4. Not a big hangup, but worth checking if we haven't already.